csharp/autogen/src/JsonRpc.cs

```diff
@@ -257,7 +258,31 @@ private T Rpc<T>(string callName, JToken parameters, JsonSerializer serializer)
                                 string json2 = responseReader.ReadToEnd();
                                 var res2 = JsonConvert.DeserializeObject<JsonResponseV2<T>>(json2, settings);
 #else
-                                var res2 = (JsonResponseV2<T>)serializer.Deserialize(responseReader, typeof(JsonResponseV2<T>));
+                                JsonResponseV2<T> res2;
+
+                                if (callName == "VM.get_record")
+                                {
+                                    string json2 = responseReader.ReadToEnd();
+
+                                    var requestProperties = TypeDescriptor.GetProperties(webRequest);
+                                    var requestSb = new StringBuilder("\n*** WebRequest Properties:\n");
+                                    foreach (PropertyDescriptor prop in requestProperties)
+                                        requestSb.AppendFormat("{0} {1}={2}\n", prop.PropertyType, prop.Name, prop.GetValue(webRequest));
+
+                                    var responseProperties = TypeDescriptor.GetProperties(webRequest);
+                                    var responseSb = new StringBuilder("*** WebResponse Properties:\n");
+                                    foreach (PropertyDescriptor prop in responseProperties)
+                                        responseSb.AppendFormat("{0} {1}={2}\n", prop.PropertyType, prop.Name, prop.GetValue(webRequest));
```

Comment:
I know this PR needs updating however this issue seems unrelated to the upcoming update.

You're using `TypeDescriptor.GetProperties` with `webRequest`, but trying to list properties of the web response. This will create an almost identical string between `responseSb` and `requestSb`.

 Not sure this was on purpose.

Reply:
More like a stupid copy-paste error... 😵